src/path.ts

```diff
@@ -193,24 +221,47 @@ export function normalizeString(path: string, allowAboveRoot: boolean) {
   return res;
 }
 
-// isAbsolute
+/**
+ * Determines if a path is an absolute path.
+ *
+ * @param p - The path to check.
+ * @returns `true` if the path is absolute, otherwise `false`.
+ */
 export const isAbsolute: typeof path.isAbsolute = function (p) {
   return _IS_ABSOLUTE_RE.test(p);
 };
 
-// toNamespacedPath
+/**
+ * Converts a non-namespaced path into a namespaced path. On POSIX systems this is a noop.
+ *
+ * @param p - The path to convert.
+ * @returns the namespaced path.
+ */
 export const toNamespacedPath: typeof path.toNamespacedPath = function (p) {
```

Comment:
jsdocs are inferred from node types in standard utils:

<img width="805" alt="image" src="https://github.com/user-attachments/assets/c48405ff-3b35-4222-9a31-855af44c236a" />